cookbook/session/locale_sticky_session.rst

```diff
@@ -106,3 +106,86 @@ method::
     {
         $locale = $request->getLocale();
     }
+
+Setting the locale based on the user entity
+-------------------------------------------
+
+You might want to improve even further and want to define the locale based on
+the user entity of the logged in user. However since the `LocaleListener` is called
+before the `FirewallListener`, which is responsible for handling authentication and
```

Comment:
you have to use double backticks in reStructed Text (same in the rest of your changes)